java/androidpayload/library/pom.xml

```diff
@@ -59,12 +59,21 @@
 				<artifactId>android-maven-plugin</artifactId>
 				<configuration>
 					<sdk>
-						<!-- platform or api level (api level 10 = platform 2.3)-->
-						<platform>10</platform>
+						<!-- platform or api level (api level 19 = platform 4.4 (kitkat))-->
```

Comment:
4.1.1.4 is the maven plugin version, 4.4 is the android sdk version which is OK here.